src/components/Button/Button.style.ts

```diff
@@ -316,11 +329,17 @@ function buttonVariants({
         &:hover {
           border: ${borderWidth} solid transparent;
           background: ${rgba(color, 0.1)};
-          /* color: hoverColorDark */
           color: ${hoverColor};
         }
+
+        ${active && minimalTransparentActiveCSS}
       `;
     case 'transparent':
+      const transparentActiveCSS = css`
+        background: ${rgba(shade(0.2, activeColor), 0.75)};
+        transform: scale(1);
```

Comment:
This should be shared with the active psuedoclass. Also, the current active psuedoclass has a different scale so those need to be aligned.

Reply:
done and aligned without scale